.github/workflows/pre_release.yaml

```diff
@@ -0,0 +1,105 @@
+name: Create a pre-release
+
+on:
+    # Push to master will deploy a beta version
+    push:
+        branches:
+            - master
+            # This will allow releasing beta versions from non-master releases.
+            # TODO: We need to release this version with other tags than beta. It can mess up the beta version as it will override it with older code.
+            - version/**
+
+jobs:
+    release_metadata:
+        if: "!startsWith(github.event.head_commit.message, 'docs') && !startsWith(github.event.head_commit.message, 'ci')"
```

Comment:
Lets add in a check that it runs on our org (so forked users dont have this try to needlessly run)

Reply:
Not sure I understand, but I admit I didn't consider forks too hard. Can you elaborate please?

Reply:
Basically adding `github.repository_owner == 'apify'` in the if-check

Reply:
Right, because when somebody makes a fork, it might contain a master branch and you don't want to release from that. Got it.

Reply:
Hmm, do we deal with this anywhere else? I don't think so, and I've never heard of some failed release pipeline from a fork, are we sure this is valid problem? Either way, the fork won't have secrets, so we don't have to be afraid of that much, but I wonder if it can actually happen.

Reply:
Its mostly so we don't needlessly run steps that will fail anyways on forks (all 5 of them or however many we have :D)

Reply:
I don't follow, what 5 forks we have?

My question still holds, where are we dealing with this? Because I am not aware of a single workflow where we would check the org name.

I am not saying its a bad idea, but it feels weird that it only pops now, if it would be an actual problem, how come we haven't seen that years ago?

Reply:
I'm not sure that we did pre-releases on push to master before - that might be the reason we're only dealing with this now.

Reply:
True, but the previous version was adding git tags for each canary build, which isn't that different.

Anyway, let's do it, although I would love to see a better solution than hardcoding org name (but lets not waste much time on this).

Reply:
You don't need NPM credentials to make a git tag locally though :slightly_smiling_face: Also I'm not sure which name the default branch for a new fork receives, but I'd wager it won't be `master`.

All aside, just checking if this happens under `apify/` is just fine IMO.